price-feeder/config/currency_provider_tracker.go

```diff
@@ -111,6 +111,11 @@ func (t *CurrencyProviderTracker) setCoinIDSymbolMap() error {
 // support each price feeder currency pair and store it in the CurrencyProviders map.
 func (t *CurrencyProviderTracker) setCurrencyProviders() error {
 	for _, pair := range t.pairs {
+		// OSMO/ATOM is supported by UMEE's osmosis-api.
+		if pair.Base == "OSMO" && pair.Quote == "ATOM" {
+			t.CurrencyProviders[pair.Base] = append(t.CurrencyProviders[pair.Base], "osmosisv2")
+		}
+
```

Comment:
So will we have to do this every time we add another asset to the osmo API?

Reply:
Only if that market does not exist on coin gecko

Reply:
@rbajollari why does this market not exist on coingecko? Does it only have "ATOM/OSMO"?

We can just allow the flip of these too for the websocket API (I.e. if ATOM/OSMO exists, then OSMO/ATOM also exists)

Reply:
Should I address this in another PR since we talked about exposing what assets are supported on our osmo API through an endpoint on it?

Reply:
@rbajollari yeah let's do this in another PR after the websocket adds an endpoint for it